### PR TITLE
Change all tokens link to engine by REST controllers

### PR DESCRIPTION
**Pull request by ifochka9519**
Improve code. Change all ModelAndView Controllers to REST Controllers

### CODE REVIEW

File: src/main/resources/templates/invitations/form.html [hunk 1]
```diff
@@ -52,26 +44,44 @@ <h3 id="formTitle" class="mb-30">INVITATION FORM</h3>
...
-                "email" : email,
                 "username" : username,
-                "firstName" : firstName,
-                "lastName" : lastName,
                 "password" : password
             };
 
         $.ajax({
-            type: "post",
+            type: "get",
```
Comment (inventorsoft-academy) [inline src/main/resources/templates/invitations/form.html:84]: why get?

File: src/main/resources/templates/invitations/form.html [hunk 2]
```diff
@@ -52,26 +44,44 @@ <h3 id="formTitle" class="mb-30">INVITATION FORM</h3>
...
             url: "/api/users/users/user",
```
Comment (inventorsoft-academy) [inline src/main/resources/templates/invitations/form.html:85]: /users is duplicated

File: src/main/resources/templates/passwords/forgotPassword.html [hunk 3]
```diff
@@ -41,6 +41,32 @@ <h3 id="formTitle" class="mb-30">INPUT YOUR NEW PASSWORD</h3>
 <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
```
Comment (inventorsoft-academy) [inline src/main/resources/templates/passwords/forgotPassword.html:41]: replace with local version

**Review by inventorsoft-academy — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/invitations/form.html (hunk 1 above)
Comment (ifochka9519) [reply src/main/resources/templates/invitations/form.html:84]: I doubted what right, I'll change.

File: src/main/resources/templates/invitations/form.html (hunk 2 above)
Comment (ifochka9519) [reply src/main/resources/templates/invitations/form.html:85]: I'll fix!

File: src/main/resources/templates/passwords/forgotPassword.html (hunk 3 above)
Comment (ifochka9519) [reply src/main/resources/templates/passwords/forgotPassword.html:41]: Ok!

**Pull request merged**